fxmanifest.lua

```diff
@@ -18,7 +18,8 @@ client_scripts {
     '@PolyZone/EntityZone.lua',
     '@PolyZone/CircleZone.lua',
     '@PolyZone/ComboZone.lua',
```

Comment:
Only `CircleZone` is used, no?
I see no usage of `EntityZone`, `BoxZone`, `ComboZone` atleast

Reply:
maybe?
it has been a good while since i have used PolyZone, so didnt want to break it lol

Reply:
> then they should probably be removed to improve performance / overhead, no?

indeed. will do some testing and see if the others are required :)